Sara/mkt 500 re rebrand the docs with new product (#878)
returntocorp/semgrep-docs: docs/writing-rules/private-rules.md

---
slug: private-rules

> description: "Users in the Team and Enterprise tier for Semgrep Cloud Platform can publish rules to the Semgrep Registry that are not visible to others outside their organization. This can be useful for organizations where rules may contain code-sensitive information or legal requirements prevent using a public registry."

Reviewer: ```suggestion
description: "Users of the Team and Enterprise tier for Semgrep Code can publish rules to the Semgrep Registry that are not visible to others outside their organization. This can be useful for organizations where rules may contain code-sensitive information or legal requirements prevent using a public registry."
```

---

import MoreHelp from "/src/components/MoreHelp"

# Private rules

> Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep Cloud Platform can publish rules to the Semgrep Registry as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.

Reviewer: ```suggestion
Users in the [Team and Enterprise tier](https://semgrep.dev/pricing) for Semgrep Code can publish rules to the Semgrep Registry as private rules that are not visible to others outside their organization. The private rules enable you to hide code-sensitive information or legal requirements that prevent you from using a public registry.
```
